storage/engine/db.cc

```diff
@@ -33,6 +33,10 @@
 #include "encoding.h"
 
 extern "C" {
+#include "_cgo_export.h"
+}  // extern "C"
```

Comment:
Just curious: Is it significant that this is in a separate `extern "C"` block from what follows?